### PR TITLE
add support for test on external oidc cluster

### PR DESCRIPTION
**Pull request by yanpzhan**
@fxierh @yapei pls help to review.

### CODE REVIEW

File: ci-operator/step-registry/openshift-extended/web-tests/openshift-extended-web-tests-commands.sh [hunk 1]
```diff
@@ -17,7 +17,17 @@ fi
...
   echo "Testing on hypershift guest cluster"
-  ./console-test-frontend-hypershift.sh || true
+  if (oc get authentication cluster -o=jsonpath={.spec.type} --kubeconfig=${KUBECONFIG} | grep OIDC);then
+    echo "Testing on Cluster using external OIDC"
+    export USER_EMAIL=$(cat /var/run/hypershift-qe-ci-rh-sso-service-account/email)
+    export USER=$(cat /var/run/hypershift-qe-ci-rh-sso-service-account/rh-username)
+    export PASSWORD=$(cat /var/run/hypershift-qe-ci-rh-sso-service-account/rh-password)
+    export issuer_url=$(cat /var/run/hypershift-ext-oidc-cli/issuer-url)
+    export cli_client_id=$(cat /var/run/hypershift-ext-oidc-cli/client-id)
+    ./console-test-azure-external-oidc.sh || true
```
Comment (fxierh) [inline ci-operator/step-registry/openshift-extended/web-tests/openshift-extended-web-tests-commands.sh:27]: Could we allow the propagation of the exit status ?  
```suggestion
    ./console-test-azure-external-oidc.sh
```

Comment (fxierh) [reply ci-operator/step-registry/openshift-extended/web-tests/openshift-extended-web-tests-commands.sh:27]: Also, is it possible to invoke `set -e` somewhere, so `./console-test-azure-external-oidc.sh` failures causes the entire step to fail (and later test steps will not run) ?

**Comment by yanpzhan** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-azure-ipi-ovn-hypershift-guest-f7

**Comment by yanpzhan** [comment]
@kuiwang02 do you know how to add rehearsals-ack label so that the pr could merge?

**Pull request merged**